src/pages/settings/Subscription/CardSection/BillingBanner/PreTrialBillingBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Report from '@libs/actions/Report';
+import Navigation from '@libs/Navigation/Navigation';
+import * as ReportUtils from '@libs/ReportUtils';
+import ROUTES from '@src/ROUTES';
+import BillingBanner from './BillingBanner';
+
+function PreTrialBillingBanner() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const navigateToChat = () => {
+        const reportUsedForOnboarding = ReportUtils.getChatUsedForOnboarding();
+
+        if (!reportUsedForOnboarding) {
+            return;
```

Comment:
When `reportUsedForOnboarding` is unavailable, an early return without navigation is not a good user experience considering that the user wanted to complete the setup checklist. 

Also, we have a usage of navigation to System/Concierge Chat [here](https://github.com/Expensify/App/blob/75614394a2fb1e8114e35bb2d8d33bdd0c565946/src/pages/OnboardingPersonalDetails/BaseOnboardingPersonalDetails.tsx#L75C1-L79C14). Here again, there will be no navigation if the System Chat report is unavailable which does not sound correct. 

I think a utility function will help here which can be used in both these places. And in the event of unavailable report id, we can fallback to navigation to concierge chat using `navigateToConciergeChat`.  
What do you think?

Reply:
I put that condition for safety purposes but do you think such scenario is even possible to happen? From my understanding users will always have access to both system and concierge chats, so the early return would probably never happen.

Reply:
That’s interesting because I notice that these two chats are present immediately after a new user signs up. So, you are right that the scenario would not happen.
But then it is good to check for invalid report id as that would bring about `Hmm… it’s not here` page which is not good. 
If at all the condition occurs, can we navigate to concierge page using `navigateToConciergeChat` and then return?
At least, we are safe that way.

Reply:
Adjusted!